src/k8s_api_endpoints.py

```diff
@@ -38,8 +38,8 @@ def get_external_api_endpoint(self) -> Optional[str]:
         return response.address
 
     def get_internal_api_endpoint(self) -> Optional[str]:
-        """Endpoint address from the loadbalancer-external relation."""
-        response = self.charm.lb_external.get_response("api-server-internal")
+        """Endpoint address from the loadbalancer-internal relation."""
+        response = self.charm.lb_internal.get_response("api-server-internal")
```

Comment:
INTERNAL loadbalancer IP should come from the INTERNAL relation, not the external relation